src/main/kotlin/loan/GuaranteedLoans.kt

```diff
@@ -0,0 +1,15 @@
+package loan
+
+import customer.Customer
+
+class GuaranteedLoans() : Loanable {
+
+    override fun verifyLoan(customer: Customer): Boolean {
+        val minIncome = 3000.0
+        val maxIncome = 5000.0
+        return (customer.location == "SP") && (customer.income > minIncome && customer.income < maxIncome)
```

Comment:
Humm quase isso André rsrsrs
As regras de empréstimo com Garantia são beem mais complicadas :eyes: 
```suggestion
        if (customer.income <= 3000)
            return customer.age < 30
        elseif (customer.income < 5000)
            return customer.localization == "SP"
        else
            return (customer.age < 30 && customer.localization == "SP")
```
Do jeito que está o seu código, ele só está validando 1 das regras de validação para ser elegivel ao empréstimo com garantia :eyes: 
![image](https://github.com/andreneery/code-challenge/assets/53612103/29b4e31b-a57e-4593-92da-7ee09e6128bf)

Como não é o foco do exercício, pode depois só alterar essa classe com os ifs que sugeri :+1: